inc/wp-components/classes/class-pagination.php

```diff
@@ -42,18 +42,14 @@ public function default_config() : array {
 	 * @return self
 	 */
 	public function query_has_set() : self {
-
 		// Get the pagination links for the query.
 		$pagination_links = $this->get_pagination_links();
 
 		// Convert each HTML link to a Pagination_Item.
 		if ( ! empty( $pagination_links ) ) {
-
 			foreach ( $pagination_links as $link_html ) {
 				$this->append_child(
-
-					// Create a new pagination item using anchor HTML, and remove
-					// various url params.
+					// Create a new pagination item using anchor HTML, and remove// various url params.
```

Comment:
Was this change intended as part of this pull request? If so, should probably remove the `\\`

Reply:
Thanks - I'll remove it. I was fixing PHPCS issues - but missed that comment tag.